assets/scss/app.scss

```diff
@@ -1,8 +1,7 @@
 $path: "../img/";
 
-@import "hof-frontend-toolkit";
+@import "hof-theme-govuk";
 
-@import "./alert.scss";
 @import "./icons.scss";
 @import "./validation.scss";
 @import "./print.scss";
```

Comment:
They might be, I don't actually know. 

I just know that this particular PR doesn't make them any more common than they previously were.

`alerts.scss` was definitely rolled into `hof-frontend-toolkit` a few weeks ago (by me) so I removed that one, but I don't know the status of the others. I'll take a look.